conference/management/commands/create_initial_data_for_dev.py

```diff
@@ -75,7 +76,13 @@ def handle(self, *args, **options):
             talk.created_by = user.user
             talk.save()
             add_speaker_to_talk(speaker, talk)
-            get_or_create_attendee_profile_for_new_user(user.user)
+            profile = get_or_create_attendee_profile_for_new_user(user.user)
+            if not profile.getBio():
```

Comment:
I'm not sure why this works, because  `get_or_create` should create a dummy Bio (`bio`) on DoesNotExist, here: https://github.com/EuroPython/epcon/blob/0f43d49fef50160e8c83edc4eb0419eff69fcd6a/conference/accounts.py#L161

Reply:
The AttendeeProfiles are already created at this point - when creating AssopyUser, `user_created` is triggered and calls `p3.listeners.on_user_created` which calls `AttendeeProfile.objects.getOrCreateForUser(sender.user)`.

Reply:
hmm... I remember disabling this signal somewhere... maybe on some un-merged yet branch. Another reason to make sure we merge things ASAP rather than let them live for too long. ;)